src/Tests/SDDLTests/Program.cs

```diff
@@ -0,0 +1,323 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Versioning;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using System.Text.RegularExpressions;
+using Microsoft.DotNet.Installer.Windows;
+using Xunit;
+
+namespace SDDLTests
+{
+    /// <summary>
+    /// Console application containing a manual test to verify security descriptors used by workloads when caching packages.
+    /// </summary>
+    [SupportedOSPlatform("windows")]
```

Comment:
I glanced at this only briefly, and sorry for a newb question 🙂

Why does this have to be windows-specific? I noticed a few things that seemed to be the windows flavor of a more general concept (like "IsAdministrator" using windows-specific APIs), but I think this should ideally work across platforms, which means it should ideally be tested across platforms, right?

Reply:
I believe that's because it's for the MSI Package cache, and MSIs are a windows only concept

Reply:
Follow-up: What's MSI? I thought it was Microsoft Installer, presumably for the SDK, which is cross-platform, so I think I have a broken link in my brain.